### PR TITLE
Properly destroy groupchats on lock/gban

### PR DESCRIPTION
**Pull request by whalemer**

### CODE REVIEW

**Comment by whalemer** [comment]
Should this be using ``Users.HOST_SYMBOL``?

**Comment by Zarel** [comment]
Yes, good call.

File: chat-commands.js [hunk 1]
```diff
@@ -1787,7 +1787,7 @@ const commands = {
 			Rooms.rooms.forEach((curRoom, id) => {
 				if (id === 'global' || !curRoom.auth) return;
 				// Destroy personal rooms of the locked user.
-				if (curRoom.isPersonal && curRoom.auth[userid] === '#') {
+				if (curRoom.isPersonal && curRoom.auth[userid] === '\u2605') {
```
Comment (Zarel) [inline chat-commands.js]:
```suggestion
				if (curRoom.isPersonal && curRoom.auth[userid] === Users.HOST_SYMBOL) {
```

File: chat-commands.js [hunk 2]
```diff
@@ -1992,7 +1992,7 @@ const commands = {
 		for (const roomid of targetUser.inRooms) {
 			if (roomid === 'global') continue;
 			let targetRoom = Rooms.get(roomid);
-			if (targetRoom.isPersonal && targetRoom.auth[userid] === '#') {
+			if (targetRoom.isPersonal && targetRoom.auth[userid] === '\u2605') {
```
Comment (Zarel) [inline chat-commands.js]:
```suggestion
			if (targetRoom.isPersonal && targetRoom.auth[userid] === Users.HOST_SYMBOL) {
```

**Pull request merged**